public/assets/css/main.css

```diff
@@ -1083,7 +1083,7 @@ h6,
     font-size: 14px;
     font-weight: 500;
     color: #262d3f;
-    display: block;
+    display: inline;
```

Comment:
Isn't this the plainadmin theme's file? Maybe we want to override their rule with our own css instead of rewriting their theme

Reply:
Reverted